blog-content/error-monitoring-launch-week-2-new-features.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Error Monitoring - Auto Resolver, Extension Filtering, and the Instance View
+createdAt: 2023-07-17T12:00:00Z
+readingTime: 2
+authorFirstName: Eric
+authorLastName: Thomas
+authorTitle: Software Engineer
+authorTwitter: ''
+authorLinkedIn: 'https://www.linkedin.com/in/eric-l-m-thomas/'
+authorGithub: 'https://github.com/et'
+authorWebsite: 'https://elmthomas.com/'
+authorPFP: 'https://www.highlight.io/_next/image?url=https%3A%2F%2Fmedia.graphassets.com%2FnGV2kef5QWrrpTb8zSx3&w=3840&q=75'
+tags: Launch Week 2 
+metaTitle: Error Monitoring - Auto Resolver, Extension Filtering, and the Instance View
+---
+
+Getting your application’s errors count down is a never ending battle. The enlightened developer would want the equivalent of Inbox Zero for errors but practically that’s a pipe dream. Errors come in at an inconsistent rate and context switching to fix them is a heavy toll. What’s worse is when you spend an exorbitant amount of time on an error to uncover that it’s completely unrelated to your code.
+
+At Highlight, with our commitment to dogfood the product  ourselves, we found that our error monitoring made it hard to tame errors on a consistent basis. While we speculated on various reasons why, we landed on one root cause: too much noise and not enough signal.
+
+Today we are launching several new features to help make every developer’s error monitoring experience more delightful and surface only what’s most important.
+
+## Auto resolving errors
```

Comment:
Add something like this to add to the end of the section:

"To enable this feature, head over to your [project settings](https://app.highlight.io/settings/errors) and enable the "auto-resolve stale errors" feature. And read about all the options for managing errors in [our docs](https://www.highlight.io/docs/general/product-features/error-monitoring/managing-errors#autoresolving-errors).